.github/workflows/test.yaml

```
@@ -0,0 +1,74 @@
+---
+name: Lint & Test
+
+# Run for all pushes to main and pull requests when Go or YAML files change
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.19'
```

```suggestion
          go-version: '1.20'
```
Should we upgrade to 1.20 now?